src/components/Footer/Footer.module.css

```diff
@@ -1,4 +1,10 @@
 footer {
+  width: 100vw;
+  display: flex;
+  justify-content: space-between; /* Aligns content on either side */
```

Comment:
personally I would remove the comments here on the css - they styles we're adjusting here are pretty common so I don't think they benefit from the extra explanation.